src/parser/utils.ts

```diff
@@ -27,3 +27,7 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+export type InspectParameterFn = (x: unknown, options: unknown) => string;
+// eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+export const defaultInspect: InspectParameterFn = v => `${v}`;
```

Comment:
This default inspect is not good enough to handle types that do not stringify well, notably, objects will become `"[object Object]"`. We know `JSON.stringify` has limitations but I think we should strive to get a string that is representative of the data.